routes/fire.py

```diff
@@ -1,96 +1,27 @@
-    12: {"type": "Sub-polar or polar grassland-lichen-moss", "color": "#bad48f"},
-    13: {"type": "Sub-polar or polar barren-lichen-moss", "color": "#408a70"},
-    14: {"type": "Wetland", "color": "#6ba38a"},
-    15: {"type": "Cropland", "color": "#e6ae66"},
-    16: {"type": "Barren land", "color": "#a8abae"},
-    17: {"type": "Urban and built-up", "color": "#DD40D6"},
-    18: {"type": "Water", "color": "#4c70a3"},
-    19: {"type": "Snow and ice", "color": "#eee9ee"},
-}
-smokey_bear_names = {
-    1: "Low",
-    2: "Medium",
-    3: "High",
-    4: "Very High",
-    5: "Extreme",
-    6: "No data at this location.",
-}
-smokey_bear_styles = {
-    1: "#2b83ba",
-    2: "#abdda4",
-    3: "#ffffbf",
-    4: "#fdae61",
-    5: "#d7191c",
-    6: "#ffffff",
-}
-snow_status = {
-    1: "Sea",
-    2: False,
-    3: "Sea ice",
-    4: True,
-    0: "No data at this location.",
-}
-
-
-async def fetch_layer_data(url, session):
-    """Make an awaitable GET request to URL, return json"""
-    resp = await session.request(method="GET", url=url)
-    resp.raise_for_status()
-    json = await resp.json()
-    return json
-
-
-async def fetch_fire_data(lat, lon):
-    """Fire API - gather all async requests for fire data"""
-    bbox_offset = 0.000000001
-    # base urls should work for all queries of same type (WMS, WFS)
-
-    base_wms_url = (
-        GS_BASE_URL
-        + f"alaska_wildfires/wms?SERVICE=WMS&VERSION=1.1.1&REQUEST=GetFeatureInfo&FORMAT=image%2Fjpeg&TRANSPARENT=true&QUERY_LAYERS=alaska_wildfires%3A{{0}}&STYLES&LAYERS=alaska_wildfires%3A{{0}}&exceptions=application%2Fvnd.ogc.se_inimage&INFO_FORMAT=application/json&FEATURE_COUNT=50&X=1&Y=1&SRS=EPSG%3A4326&WIDTH=1&HEIGHT=1&BBOX={lon}%2C{lat}%2C{float(lon) + bbox_offset}%2C{float(lat) + bbox_offset}"
-    )
-
-    base_wfs_url = (
-        GS_BASE_URL
-        + f"alaska_wildfires/wfs?SERVICE=WFS&VERSION=1.1.0&REQUEST=GetFeature&TypeName={{}}&PropertyName={{}}&outputFormat=application/json&srsName=urn:ogc:def:crs:EPSG:4326&BBOX={lat}%2C{lon}%2C{float(lat) + bbox_offset}%2C{float(lon) + bbox_offset}%2Curn:ogc:def:crs:EPSG:4326"
-    )
-
-    urls = []
-    # append layer names for URLs
-    urls.append(base_wms_url.format("alaska_landcover_2015"))
-    urls.append(base_wms_url.format("spruceadj_3338"))
-    urls.append(base_wms_url.format("snow_cover_3338"))
-    urls.append(
-        base_wms_url.format("alfresco_relative_flammability_NCAR-CCSM4_rcp85_2000_2099")
-    )
-    urls.append(base_wfs_url.format("historical_fire_perimiters", "NAME,FIREYEAR"))
-
-    async with ClientSession() as session:
-        tasks = [fetch_layer_data(url, session) for url in urls]
-        results = await asyncio.gather(*tasks)
-    return results
+wms_targets = [
+    "alaska_landcover_2015",
+    "spruceadj_3338",
+    "snow_cover_3338",
+    "alfresco_relative_flammability_NCAR-CCSM4_rcp85_2000_2099",
+]
+wfs_targets = {"historical_fire_perimiters": "NAME,FIREYEAR"}
```

Comment:
This is how the name of the data layer is spelled in Geoserver so I'll have to leave it as is. We could change it in GS and then change it here but it might break other stuff that pulls data from that layer.

Reply:
Ahh okay! Sounds good we can look at that another day then when all apps rely exclusively on the API.